Frameworks/UIKit/UINibUnarchiver.mm

```diff
@@ -203,7 +203,13 @@ static id constructObject(UINibUnarchiver* self, Object* pObj) {
         id classId = pObj->classType; // objc_getClass(pObj->className);
         assert(classId != nil);
 
-        pObj->cachedId = [classId alloc];
+        // TODO: #1105 alloc without init is invalid, and not all classes support alloc-init anyway
+        // look for a proper fix to this issue, rather than just special-casing for UIFont
```

Comment:
>proper [](start = 22, length = 6)

Not sure if this is an issue, we already have several special cases in this method.